From f6b882d06fdf882781dd9f386298f270a05729c5 Mon Sep 17 00:00:00 2001
From: Benoit, NicolasX <nicolasx.benoit@intel.com>
Date: Fri, 9 Nov 2012 16:13:56 +0100
Subject: [PORT FROM MAIN] dumpstate: Add mid_pmu_states dump in android bugreport

BZ: 21478

Change-Id: I2dbbf992b8b871ef3f7aac1358d1e6dcf83cb6f9
Orig-Change-Id: I634007eb363ee90bd55ee027481883b7aa2c8780
Reviewed-on: http://android.intel.com:8080/80736
Reviewed-by: Popescu, Catalin <catalin.popescu@intel.com>
Reviewed-by: Li, XiaojingX <xiaojingx.li@intel.com>
Tested-by: Li, XiaojingX <xiaojingx.li@intel.com>
Reviewed-by: cactus <cactus@intel.com>
Tested-by: cactus <cactus@intel.com>
---
 cmds/dumpstate/dumpstate.c |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/cmds/dumpstate/dumpstate.c b/cmds/dumpstate/dumpstate.c
index db3f250..756e0ae 100644
--- a/cmds/dumpstate/dumpstate.c
+++ b/cmds/dumpstate/dumpstate.c
@@ -89,6 +89,7 @@ static void dumpstate() {
     dump_file("BUDDYINFO", "/proc/buddyinfo");
     dump_file("FRAGMENTATION INFO", "/d/extfrag/unusable_index");
 
+    dump_file("PMUSTATES", "/sys/kernel/debug/mid_pmu_states");
 
     dump_file("KERNEL WAKELOCKS", "/proc/wakelocks");
     dump_file("KERNEL WAKE SOURCES", "/d/wakeup_sources");
-- 
1.7.4.1

